spec/system/admin_company_value_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'CompanyValue check', type: :system do
+  before do
+    driven_by(:rack_test)
+  end
+
+  let!(:admin_user) { create(:admin_user) }
+  let!(:company_value) { build(:company_value) }
+
+  it 'crud CompanyValue' do
+    login_as(admin_user)
+    visit admin_root_path
+
+    click_link 'Company value'
+    click_link 'New Company value'
+    fill_in 'Title', with: company_value.title
+    click_button 'Create Company value'
+    expect(page).to have_content 'Company value was successfully created.'
+    expect(page).to have_content 'title company'
```

Comment:
if we are filling title with variable `company_value.title`, then we should expect to have variable `company_value.title`, not fixed string
Right now if you change title in factory for company_value, it will fail this test on line 22

```suggestion
    expect(page).to have_content company_value.title
```

Reply:
you are right, one mistake less just in case :)